songexile/cise_assignment_1b
# Miguel/feature/moderator

👨🏻‍💻 What does this PR do?

This PR is to complete Moderator user stories

🧪 How has/should this change been tested?
Manual Test

📝 Any notes for the reviewer?

📜 Documentation
New App Flow

```
Anyone
-> submit an article
-> data goes to moderator DB
-> moderator get data from moderator DB to check quality
-> moderator accept
-> data goes to analyst db
-> analyst get data from analyst DB to accept or decline
-> analyst accept
-> data goes to speed db
-> anyone now can see articles with claim and method from speed db

Total db 3 (moderator, analyst, speed)
```
